frontend/assets/pwa-manifest.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
I wanted to prepare as many icon sizes as possible here (used [this online tool](https://www.iloveimg.com/resize-image/resize-png) to generate multiple dimension icons, in case you wonder), but let me know if we want to reduce the numbers here. (for example, [MDN](https://developer.mozilla.org/en-US/) website only has two icon entries 192x192 and 512x512)

Reply:
I can't remember now, but I believe 'just' 192×192 and 512×512 is fine for all cases where this might be used. _But_, you have various types of favicons that could be set (e.g., for Apple and Microsoft browsers / OSes). I suggest taking a look here: https://realfavicongenerator.net/

Reply:
@corrideat That online tool seems pretty useful.

I'm not sure about preparing multiple icon types here though.
I haven't actually seen any docs or articles saying the icons should be prepared as multiple types yet and also, the live PWA websites I used as a reference (e.g. [MDN](https://developer.mozilla.org/en-US/), https://www.stuff.co.nz/ etc.) only have `.png` files specified for their icons. 

below is the `manifest.webmanifest` screenshot of [stuff.co.nz](https://www.stuff.co.nz/), which is New Zealand's largest news webzine.
<img src='https://github.com/okTurtles/group-income/assets/17641213/b4ff8d14-a014-4ce3-a984-705ce7a88a89' width='380'>

I think it should be fine to have two `.png` (192x192, 512x512) and a `.svg`